bedrock/products/views.py

```diff
@@ -97,14 +88,19 @@ def vpn_landing_page(request):
 @require_safe
 def vpn_pricing_page(request):
     template_name = "products/vpn/pricing-refresh.html" if request.locale == "en-US" else "products/vpn/pricing.html"
-    ftl_files = ["products/vpn/landing", "products/vpn/shared"]
+    ftl_files = ["products/vpn/landing", "products/vpn/shared", "products/vpn/pricing-2023"]
     available_countries = settings.VPN_AVAILABLE_COUNTRIES
     country = get_country_from_request(request)
     vpn_available_in_country = vpn_available(request)
     attribution_available_in_country = country in settings.VPN_AFFILIATE_COUNTRIES
     vpn_affiliate_attribution_enabled = vpn_available_in_country and attribution_available_in_country and switch("vpn-affiliate-attribution")
     variant = request.GET.get("v", None)
 
+    if ftl_file_is_active("products/vpn/pricing-2023"):
+        template_name = "products/vpn/pricing-refresh.html"
+    else:
+        template_name = "products/vpn/pricing.html"
+
     # ensure variant matches pre-defined value
     if variant not in ["1", "2"]:
         variant = None
```

Comment:
Question: do we still need the `variant` logic for this view?

Reply:
Seems like we don't -- removing. Thanks for the eagle eyes!